### ISSUE TITLE
[Feature Request] general `action : policy` syntax for all controllers

### ISSUE THREAD

**Issue by mphasize** [balderdashy/sails#2713]
The policies config file currently supports `'*'` for all Controllers or `controller: { ... options ... }`. 
It would be super nice, if we could set a policy on a certain blueprint action across all controllers, just like:

```javascript
module.exports.policies = {
  create : 'beforeCreate'
};
```

If that is too hard to parse and differentiate between actions and controllers, another nice syntax might be `'*.create' : 'beforeCreate'`.

**Comment by sgress454** [comment]
Hi @mphasize, great minds think alike; there's already something about this [on the roadmap](https://github.com/balderdashy/sails/blob/master/ROADMAP.md).

**Comment by mphasize** [comment]
@ProLoser I saw that you're the owner of this idea in the Sails backlog. Would you be interested in creating a PR together?

**Comment by mphasize** [comment]
@sgress454 Sorry I didn't check the roadmap for that. I promise I searched the issues before I posted this. ;-)

**Comment by sgress454** [comment]
In the meantime you can still do this with policies--the trick is to use `config/routes.js` and the [policy target syntax](http://sailsjs.org/#!/documentation/concepts/Routes/RouteTargetSyntax.html?q=policy-target-syntax):

```
"/*/create": {policy: "myCreatePolicy"}
```

**Comment by mphasize** [comment]
:+1:

**Comment by ProLoser** [comment]
I can if you'd like although I'm not actively developing on sails right at this very moment.
